ISSUE_TEMPLATE.md

```diff
@@ -14,6 +14,21 @@ Large features typically go through three stages: [Alpha, Beta, and Stable](http
 Each stage requires various approvals from various teams.  Features require several releases
 to reach Stable.
 
+The following items have to be filled before the features freeze:
```

Comment:
I would like all of the explanatory text about the process (above and below) moved to a separate doc, with just a link here.

I would like the template populated when the issue is created, not later. First of all, that would make automatic checking by a bot easier. Second, Github does not send notifications for updates to the description. When I receive a notification about a feature, I want to see all relevant information.